app/src/features/HjelpeTekst.tsx

```diff
@@ -0,0 +1,53 @@
+import type { AlertProps, ReadMoreProps } from "@navikt/ds-react";
+import { Alert } from "@navikt/ds-react";
+import { ReadMore, Switch } from "@navikt/ds-react";
+import { useNavigate, useSearch } from "@tanstack/react-router";
+
+import { useLocalStorage } from "~/features/useLocalStorage.tsx";
+
+export const VIS_HJELPETEKSTER_KEY = "vis-hjelpe-tekster";
+
+export function HjelpeTekstMasterSwitch() {
+  const [_, setVisHjelpeTekst] = useLocalStorage(VIS_HJELPETEKSTER_KEY, true);
+  const { visHjelpeTekst } = useSearch({ from: "/$id" });
+  const navigate = useNavigate();
+
+  const onHjelpeTekstChanged = (newValue: boolean) => {
+    setVisHjelpeTekst(newValue);
+    navigate({
+      search: (prevSearch) => ({ ...prevSearch, visHjelpeTekst: newValue }),
+    });
+  };
+
+  return (
+    <Switch
+      checked={visHjelpeTekst}
+      onChange={(e) => onHjelpeTekstChanged(e.target.checked)}
+    >
+      Vis hjelpetekster i skjema
+    </Switch>
+  );
+}
+
+export function HjelpeTekst({
```

Comment:
Startet med at jeg trodde det kun var 1 komponent som skulle toggles, men så var det to. Kan argumentere for at det heller burde lages en wrapper som brukes rundt komponentene, istedenfor egne som dette 🤷